### ISSUE TITLE
wrong line highlighted / highlight shifted up

### ISSUE THREAD

**Issue by ELECTRO-BOSS** [mozilla/pdf.js#12881]
When I highlight specific text in the pdf viewer beta and copy it the wrong part gets copied instead of the highlighted text. I copy the text 2 lines under the highlight. I have the problem only with specific pdf, for example with an ISO from breuth. I tried this using Adobe Acrobat, Chrome, Edge and on a second Windows Computer. None of those pdf viewers had that problem. Maybe it is a software bug?
I made a page to test my issue from the document I had the problem with
[testpage.pdf](https://github.com/mozilla/pdf.js/files/5844543/testpage.pdf)

**Comment by seanwuapps** [comment]
Experienced with similar problem where the text layer is affected by the global css styles. Try adding 
```
.react-pdf__Page__textContent span { 
  line-height: 1em !important; 
}
```

**Comment by ELECTRO-BOSS** [comment]
I tried as described, reloaded the document but saw no change.
![grafik](https://user-images.githubusercontent.com/53611602/105698282-f97b1380-5f05-11eb-94bb-31b014e2d397.png)

**Comment by Snuffleupagus** [comment]
Fixed by PR #12896
